pages.ja/common/aws-configure.md

```diff
@@ -0,0 +1,36 @@
+# aws configure
+
+> AWS CLI の設定を管理します。
+> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>
+
+- 対話形式で AWS CLI を設定する（新しい設定の作成、または既定の更新）:
+
+`aws configure`
+
+- 対話形式で AWS CLI の名前付きプロファイルを設定する（新規プロファイルの作成、または既存プロファイルの更新）:
+
+`aws configure --profile {{profile_name}}`
+
+- 特定の設定変数の値を表示する:
+
+`aws configure get {{name}}`
+
+- 特定プロファイルの設定変数の値を表示する:
+
+`aws configure get {{name}} --profile {{profile_name}}`
+
+- 特定の設定変数の値をセットする:
+
+`aws configure set {{name}} {{value}}`
```

Comment:
```suggestion
`aws configure set {{名前}} {{価値}}`
```

Translations are missing for these placeholders. Google translation suggestions maybe have to be applied.

Reply:
@EmilySeville7cfg 
Thanks for the review!
I'll just correct the translation of "value".

Reply:
[7c5a63a](https://github.com/tldr-pages/tldr/pull/9237/commits/7c5a63a52eeb455ac067c3d6de171ebdd8102aa8)